src/lib/storage/disk_analyzer.rb

```diff
@@ -228,7 +228,7 @@ def find_windows_partitions
             disk = ::Storage::Disk.find(devicegraph, disk_name)
             disk.partition_table.partitions.each do |partition|
               if WINDOWS_PARTITION_IDS.include?(partition.id)
-                windows_partitions << partition.name if windows_partition?(partition.name)
+                windows_partitions << partition.name if windows_partition?(partition)
```

Comment:
when I see this code just small suggestion. What about putting check for `WINDOWS_PARTITION_IDS.include?` also to `windows_partition?` method? Now it is already quite deep nesting and it helps a bit.